go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/status-im/keycard-go v0.0.0-20190424133014-d95853db0f48 // indirect
 	github.com/status-im/migrate/v4 v4.3.1-status.0.20190822050738-a9d340ec8fb7
 	github.com/status-im/rendezvous v1.3.0
-	github.com/status-im/status-protocol-go v0.2.3-0.20191010161351-fd48aa752717
+	github.com/status-im/status-protocol-go v0.3.1-0.20191011120912-b115a9b0ae4f
```

Comment:
I would only try `go get github.com/status-im/status-protocol-go@v0.3.0`.

Reply:
But that will leave out the `Move `transport/whisper/gethbridge` to `bridge/geth` (#95)` commit from status-protocol-go

Reply:
```
go mod tidy
go: finding github.com/status-im/status-protocol-go/bridge/geth latest
github.com/status-im/status-go/cmd/node-canary imports
        github.com/status-im/status-protocol-go/bridge/geth: module github.com/status-im/status-protocol-go@latest (v0.3.0) found, but does not contain package github.com/status-im/status-protocol-go/bridge/geth
Makefile:290: recipe for target 'vendor' failed
make: *** [vendor] Error 1
```

Reply:
Oh right! I created https://github.com/status-im/status-protocol-go/releases/tag/v0.3.1 so it should be simply `v0.3.1` and all should work.